integration/simulation/simulation.go

```diff
@@ -115,7 +116,14 @@ func (s *Simulation) trackLogs() {
 		return
 	}
 
+	subscriptionsCreated := 0
```

Comment:
Previously, we only created a single subscription for a single wallet. I've bumped it to allowing two to be created here, since otherwise the relevancy checks in the validation are meaningless. Hopefully, once we've worked on the performance, we can give everyone a subscription.